app/views/rubygems/_rubygem.html.erb

```diff
@@ -2,6 +2,15 @@
   <span class="gems__gem__info">
     <h2 class="gems__gem__name">
       <%= rubygem.name %>
+      <% if rubygem.owners.any?(&:verified) %>
```

Comment:
For multiple owners, I chose `any` but arguably we can check if all owners are verified before marking a gem verified. Does gem ownership require manual action? I suppose that a bad user can assign a verified user co-ownership of a malicious gem and now the gem is verified.

Reply:
I think this need additional thinking. I propose to remove "verified gems" for now at all and keep it at account level.

Reply:
If we want to be on the safe side, what about just changing this to `all` instead of `any`? I think displaying "verified gems" is a core part of this feature and the main reason to merge it - It helps guard against cases where a bad actor creates a similarly named malicious gem (i.e. `aws-sdk-bad` or `activemalicious`) and markets it as official. My thoughts are that many developers search rubygems.org prior to installing gems. A future integration may be to add some `**verified**` text to gems listed on `gem search <kw>`. What do you think? 

For example, see "atlas-client" vs "atlas_client" in [this article](https://blog.reversinglabs.com/blog/mining-for-malicious-ruby-gems).